src/classes/ADDR_OtherAcc_TEST.cls

```diff
@@ -0,0 +1,204 @@
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+ 
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2016
+* @group Addresses
+* @group-content ../../ApexDocContent/Addresses.htm
+* @description Tests specific to Address Management on Accounts that are not of 
+* of the Household or Administrative types. 
+*/
+
+@isTest
+public with sharing class ADDR_OtherAcc_TEST {
+    
+    /*********************************************************************************************************
+    @description Create N new accounts with billing addresses 
+    verify:
+        N addresses created
+        account address matches address object address
+    **********************************************************************************************************/            
+    @isTest
+    public static void newRecordsWithAddrInfo() {            
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Accounts_Addresses_Enabled__c = 
+                                                                      UTIL_Describe.getBizAccRecTypeID() + ';'));
+ 
+        Integer cAcc = 3;
+        list<Account> listAcc = UTIL_UnitTestData_TEST.CreateMultipleTestAccounts(cAcc, UTIL_Describe.getBizAccRecTypeID());
+        for (Account acc : listAcc) {
+            acc.BillingStreet = '123 45th';
+            acc.BillingCity = 'Seattle';
+        }
+        Test.startTest();
+        insert listAcc;
+        Test.stopTest();
+
+        // verify results
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
+        BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude, is_Address_Override__c, Current_Address__c from Account]);
+        system.assertEquals(cAcc, mapAccIdAcc.size());
+        
+        list<Address__c> listAddr = [select Id, Default_Address__c, MailingStreet__c, MailingCity__c, Parent_Account__c, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause, or adding a `LIMIT` to contain the number of records returned.